### PR TITLE
Support HTML tags in smartling TMX downloads

### PR DESCRIPTION
**Pull request by camertron**
Causing an issue with @abMatGit's branch on staging.

@jdoconnor @zvkemp @11mdlow

### CODE REVIEW

File: lib/rosette/tms/smartling-tms/translation_memory.rb [hunk 1]
```diff
@@ -140,7 +141,13 @@ def resolve_with_placeholders(variant, placeholder_map)
           variant.elements.each_with_object('') do |el, ret|
             ret << case el
               when TmxParser::Placeholder
-                placeholder_map[el.text] || ''
+                case el.type
+                  when PLACEHOLDER_TYPE
```
Comment (jdoconnor) [inline lib/rosette/tms/smartling-tms/translation_memory.rb]: is a case statement the right approach here for a 2 decision tree?

**Comment by jdoconnor** [comment]
+:dog: with a comment

File: spec/fixtures/tmx/tmx_fixture.rb [hunk 2]
```diff
@@ -16,7 +16,9 @@ def render
     end
 
     def wrap_placeholders(text)
-      text.gsub(/(\{\d+\})/) { "<ph>#{$1}</ph>" }
+      text.gsub(/(\{\d+\})/) do
+        "<ph type=\"x-smartling-placeholder\">#{$1}</ph>"
+      end
```
Comment (abMatGit) [inline spec/fixtures/tmx/tmx_fixture.rb:21]: Ahhhh so without this, then we would just wrap all our placeholders with just `<ph>` tags regardless

Comment (abMatGit) [inline spec/fixtures/tmx/tmx_fixture.rb:21]: Is it worth to handle html placeholders as well here?

Comment (camertron) [inline spec/fixtures/tmx/tmx_fixture.rb:21]: I updated this so a test that specifically checks `{0}`-style placeholders would pass with our changes. I don't think it has anything to do with the HTML placeholders.

File: lib/rosette/tms/smartling-tms/translation_memory.rb (hunk 1 above)
Comment (camertron) [inline lib/rosette/tms/smartling-tms/translation_memory.rb]: Fixed :)

**Comment by abMatGit** [comment]
:+1: :cake:

**Pull request merged**